x-pack/plugins/observability/public/pages/alerts/components/observability_actions.tsx

```diff
@@ -32,15 +33,15 @@ import { ObservabilityRuleTypeRegistry } from '../../..';
 import { ALERT_DETAILS_PAGE_ID } from '../../alert_details/types';
 import { ConfigSchema } from '../../../plugin';
 
-export type ObservabilityActionsProps = Pick<
-  ActionProps,
-  'data' | 'eventId' | 'ecsData' | 'setEventsDeleted'
-> & {
+export interface ObservabilityActionsProps {
+  data: TimelineNonEcsData[];
```

Comment:
This more for any observability folks reading this, but I think we should move away from using this format and using the raw `SearchHit` or a more generalized agreed upon Kibana wide formatting. All this is, is a formatted fields response https://github.com/elastic/kibana/blob/main/x-pack/plugins/timelines/server/search_strategy/timeline/factory/helpers/format_timeline_data.ts#L52-L56 and I think it adds an unnecessary layer of complexity at times since there's really nothing `Timeline` specific about it

Reply:
Agree, but I don't know which type Observability team prefer here.

Reply:
@XavierM as you're the author of the `TimelineNonEcsData` interface, perhaps you have a suggestion as to what a better type here would be for Observability purposes?

Reply:
In the new alert table, we are deprecating this type but we had to keep it for security solution since a lot of functionality is working around this data/type. We need to figure out,  how we are going to fully deprecated on your side. However, o11y and stack should not use it.